includes/functions.php

```diff
@@ -1032,8 +1032,10 @@ function ap_parse_search_string( $str ) {
 				}
 			}
 
-			// Remove this pair from $bits.
-			unset( $bits[ $id ] );
+            if (isset($bits[$id])) {
```

Comment:
Can you please properly indent using tabs.
Thanks

Reply:
fixed